pyflakes/checker.py

```diff
@@ -1160,11 +1162,15 @@ def handleNodeLoad(self, node):
                 # alias of other Importation and the alias
                 # is used, SubImportation also should be marked as used.
                 n = scope[name]
-                if isinstance(n, Importation) and n._has_alias():
-                    try:
-                        scope[n.fullName].used = (self.scope, node)
-                    except KeyError:
-                        pass
+                if isinstance(n, Importation):
```

Comment:
it's not just imports -- it's any code in there

for instance:

```python
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    class C:
        ...
```

Reply:
Right, I wasn't too worried about that case, but let's af course fix that!